app/uk/gov/hmrc/customsservicestatus/models/CustomsServiceStatus.scala

```diff
@@ -65,12 +65,13 @@ object CustomsServiceStatusWithDesc {
   implicit val logger: Logger = Logger(this.getClass.getName)
   implicit val format = Json.format[CustomsServiceStatusWithDesc]
 
-  def apply(customsServiceStatus: CustomsServiceStatus): CustomsServiceStatusWithDesc = {
-    val servicesFromConfig = ConfigSource.default.loadOrThrow[ServicesFromConfig]
-    servicesFromConfig.services.find(_.name == customsServiceStatus.name) match {
-      case Some(service) => CustomsServiceStatusWithDesc(customsServiceStatus.name, customsServiceStatus.status, service.description)
-      case None          => CustomsServiceStatusWithDesc(customsServiceStatus.name, customsServiceStatus.status, "")
+  def apply(listCustomsServiceStatusFromRepo: List[CustomsServiceStatus], serviceFromConfig: ServiceFromConfig): CustomsServiceStatusWithDesc = {
```

Comment:
Can we rename this parameter from `listCustomsServiceStatusFromRepo` to  `serviceStatuses`?

Reply:
done